module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,10 @@
+#include <array>
+#include <cstdint>
+#include <vector>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(std::array<std::array<uint8_t, width>, height>& bitmap);
+std::array<std::array<uint8_t, width>, height> decompressGrayscale(std::vector<std::pair<uint8_t, uint8_t>>& compression);
```

Comment:
You can simplify this declaration for example like this
```suggestion
using Image = const std::array<std::array<uint8_t, width>, height>;
using compressedImage = std::vector<std::pair<uint8_t, uint8_t>>;
compressedImage compressGrayscale(Image& bitmap);
Image decompressGrayscale(compressedImage& compression);
```

Reply:
that should be much easier to read. thank you!